src/optimizers.ts

```diff
@@ -27,12 +27,18 @@ import {ValueError} from './errors';
 export function getOptimizer(identifier: string): Optimizer {
   const optimizerMap: {[optimizerName: string]: () => Optimizer} = {
     'Adagrad': () => train.adagrad(.01),
+    'Adadelta': () => train.adadelta(1.0, 0.95, K.epsilon()),
     'Adam': () => train.adam(.001, .9, .999, K.epsilon()),
+    'Adamax': () => train.adamax(0.002, .9, .999, K.epsilon(), 0.0),
+    'Momentum': () => train.momentum(0.01, 0.0, false),
```

Comment:
I think we want to remove the Momentum one.  All the other ones are cross-compatible with python and JS using the JSON serialization format with agreed upon symbol names.  Momentum is not in Keras at present.  But is in tensorflow & tensorflow.js core.  However the default arguments don't appear to make sense (why create a momentum optimizer with momentum set to 0?)

Reply:
I took the defaults for Momentum from https://keras.io/optimizers/ SGD Optimizer, where momentum is 0.0. 

Should I remove Momentum completely?

Reply:
Yes, please remove momentum.  This way users won't be confused if they serialized a model built in tfjs, and it then crashes in python keras.
thanks!

Reply:
thanks ! removed now Momentum